### PR TITLE
Incident report

### PR DESCRIPTION
**Pull request by aashnasheth**

### CODE REVIEW

File: src/IncidentReportView.js [hunk 1]
```diff
@@ -0,0 +1,175 @@
+import React, {Component} from "react";
+import Select from "react-select";
+import {Container, Row, Col} from 'reactstrap';
```
Comment (kushaljh) [inline src/IncidentReportView.js]: You can import this from react-bootstrap which is already used elsewhere as well

https://react-bootstrap.github.io/layout/grid/

File: src/IncidentReportView.js [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+    }
+
+    componentDidMount() {
+        this.reupdateData();
+    }
+
+    handleChange = (event) => {
+        console.log(event)
+        this.setState({
+            //value: event.label,
```
Comment (kushaljh) [inline src/IncidentReportView.js]: nit: remove

File: src/IncidentReportView.js [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+    handleBool(bool){
+        if (bool === 0){
+            return "No"
+        }
+        return "Yes"
+    }
+
+    reupdateData = () => {
+        fetch("http://localhost:4000/incidentReport/" + this.props.personID, /*{
+            body: JSON.stringify(data),
```
Comment (kushaljh) [inline src/IncidentReportView.js]: why is this in comments?

File: src/ResidentDirectory.js [hunk 4]
```diff
@@ -5,21 +5,20 @@ import AddResident from './AddResident.js'
 import Card from 'react-bootstrap/Card';
 import Button from 'react-bootstrap/Button';
```
Comment (kushaljh) [inline src/ResidentDirectory.js]: @aashnasheth 
See this for reference

I do believe that you can import multiple together as import { Card, Button } from 'react-bootstrap'; as well

File: src/ResidentDirectory.js [hunk 5]
```diff
@@ -97,11 +100,28 @@ class ResidentDirectory extends Component {
...
+                    displayCard: false,
+                    value: '?',
+                    data: '',
+                    startDate: '?'
+                });
+                this.reupdateData();
+            }
+            else if (response.status === 200) {
+                alert("lets go");
+                this.reupdateData();
```
Comment (daniellee0) [inline src/ResidentDirectory.js]: Which block is being used to handle a 200 response status?

File: src/ResidentDirectory.js [hunk 6]
```diff
@@ -39,7 +39,7 @@ class ResidentDirectory extends Component {
     }
 
     reupdateData = () => {
-        console.log("reupdating data");
+        // console.log("reupdating data");
```
Comment (kushaljh) [inline src/ResidentDirectory.js]: Maybe remove this altogether now?

File: src/ResidentDirectory.js [hunk 7]
```diff
@@ -137,7 +157,7 @@ class ResidentDirectory extends Component {
                         justifyContent: "center",
                         alignItems: "center"
                     }}>
-                        {console.log(this.state.residents)}
+                        {/* {console.log(this.state.residents)} */}
```
Comment (kushaljh) [inline src/ResidentDirectory.js]: same here

File: src/ResidentDirectory.js [hunk 8]
```diff
@@ -167,7 +187,7 @@ class ResidentDirectory extends Component {
                                 <div>
                                     <h6>Residence Information:</h6>
                                     <ul style={{listStyleType: "none"}}>
-                                        <li>Current Residence: {this.state.data.VID}; {this.state.data.RID}</li>
+                                        <li>Current Residence: house {this.state.data.HOUSE_NUM} in {this.state.data.NAME} village</li>
```
Comment (kushaljh) [inline src/ResidentDirectory.js]: Name is Village Name? 
Doesnt sound like the correct variable name.

**Review by kushaljh — CHANGES_REQUESTED** [review]

File: src/ResidentDirectory.js [hunk 9]
```diff
@@ -97,11 +100,28 @@ class ResidentDirectory extends Component {
                     .then((text) => {
                         alert(text.error);
                     });
+            } else if (response.status === 200) {
+                this.setState({
+                    displayCard: false,
+                    value: '?',
+                    data: '',
```
Comment (daniellee0) [inline src/ResidentDirectory.js:104]: - Should have a default value for "data" in constructor for clarity. 
- "data" naming is obscure.

**Review by daniellee0 — APPROVED** [review]

**Review by kushaljh — APPROVED** [review]

**Pull request merged**